velox/exec/tests/TableWriterTest.cpp

```diff
@@ -189,6 +189,66 @@ TEST_F(BasicTableWriterTestBase, targetFileName) {
       .assertResults(data);
 }
 
+TEST_F(BasicTableWriterTestBase, commitStrategyParameter) {
+  auto data = makeRowVector({makeFlatVector<int64_t>(10, folly::identity)});
+  auto directory = TempDirectoryPath::create();
+
+  // Create a plan with explicit commitStrategy parameter
```

Comment:
Can you simplify the code using a lambda function to pass different commitStrategy parameters  to test?